src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,35 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    public static final int INDEX_OF_DAY = 0;
+    public static final int INDEX_OF_NAME = 1;
+    public static final int INDEX_OF_HOURS = 2;
+    public static final int INDEX_OF_MONEY = 3;
+    public static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    public static final String enter = System.lineSeparator();
```

Comment:
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names
```suggestion
    public static final String SEPARATOR = System.lineSeparator();
```